packages/react-native/src/hooks/useOktoWebView.ts

```diff
@@ -0,0 +1,26 @@
+import { useCallback } from 'react';
+import { useNavigation } from '@react-navigation/native';
+import { useOkto } from './useOkto.js';
+
+/**
+ * Custom hook to access the OktoClient's openWebView functionality
+ *
+ * @returns A function to open web view with the provided URL
+ */
+export function useOktoWebView() {
```

Comment:
The hooks are different in both React & RN.
@karansinglacoindcx @SrijanCoinDCX please try to make them same, if possible

Reply:
@SrijanCoinDCX to update in react